app/1.0/docs/devguide/data-system.md

```diff
@@ -562,8 +562,7 @@ Related tasks:
 
 ### Element initialization
 
-When an element initializes its local DOM, it configures the properties of its local DOM children
-initializes data bindings.
+When an element initializes its local DOM, it configures the properties of its local DOM children's data bindings.
```

Comment:
Good catch. What's missing here is actually the word "and" between "configures the properties of its local DOM children **and** initializes data bindings."